packages/fluentui-rc/src/SelectWidget/SelectWidget.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  ariaDescribedByIds,
+  enumOptionsIndexForValue,
+  enumOptionsValueForIndex,
+  FormContextType,
+  labelValue,
+  RJSFSchema,
+  StrictRJSFSchema,
+  WidgetProps,
+} from '@rjsf/utils';
+import { Dropdown, Field, Option } from '@fluentui/react-components';
+import { OptionOnSelectData } from '@fluentui/react-combobox';
+
+function getValue(data: OptionOnSelectData, multiple: boolean) {
+  if (multiple) {
+    return data.selectedOptions;
+  }
+  return data.selectedOptions[0];
+}
+
+/** The `SelectWidget` is a widget for rendering dropdowns.
+ *  It is typically used with string properties constrained with enum options.
+ *
+ * @param props - The `WidgetProps` for this component
+ */
+function SelectWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>({
+  id,
+  options,
+  label,
+  hideLabel,
+  value,
+  required,
+  disabled,
+  readonly,
+  multiple = false,
+  autofocus = false,
+  rawErrors = [],
+  onChange,
+  onBlur,
+  onFocus,
+}: WidgetProps<T, S, F>) {
+  const { enumOptions, enumDisabled, emptyValue: optEmptyVal } = options;
+
+  const selectedIndexes = enumOptionsIndexForValue<S>(value, enumOptions, multiple);
+  let selectedIndexesAsArray: string[] = [];
+
+  if (typeof selectedIndexes === 'string') {
+    selectedIndexesAsArray = [selectedIndexes];
+  } else if (Array.isArray(selectedIndexes)) {
+    selectedIndexesAsArray = selectedIndexes.map((index) => String(index));
+  }
+
+  const dropdownValue = selectedIndexesAsArray
+    .map((index) => (enumOptions ? enumOptions[Number(index)].label : undefined))
+    .join(', ');
+
+  const _onBlur = () => onBlur(id, selectedIndexes);
+  const _onFocus = () => onFocus(id, selectedIndexes);
+  const _onChange = (_: any, data: OptionOnSelectData) => {
+    const newValue = getValue(data, multiple);
+    return onChange(enumOptionsValueForIndex<S>(newValue, enumOptions, optEmptyVal));
+  };
+
+  return (
+    <Field
+      // @ts-expect-error todo: TS2322: Type 'string | false | ReactElement<any, string | JSXElementConstructor<any>> | undefined' is not assignable to type 'string | undefined'.
```

Comment:
Can you write up an issue to fix this? Or you can try updating the definition of `labelValue()` as follows, which may fix it.

```
export function labelValue(label?: string, hideLabel?: boolean, fallback?: ''): undefined | string;
export default function labelValue(label?: string | ReactElement, hideLabel?: boolean, fallback?: false | '') {
  return hideLabel ? fallback : label;
}
```

Reply:
Thanks for the suggestion, it doesn't help unfortunately. I added an [issue](https://github.com/rjsf-team/react-jsonschema-form/issues/3946) and referenced to it in a comment